conda/recipes/nvstrings/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
+cd python
 $PYTHON setup.py install
-cp NVStrings.h $PREFIX/include
-cp NVCategory.h $PREFIX/include
+cp ../cpp/include/NVStrings.h $PREFIX/include
+cp ../cpp/include/NVCategory.h $PREFIX/include
 cp build/lib.linux-x86_64*/libNVStrings.so $PREFIX/lib
-cp build/lib.linux-x86_64*/libNVCategory.so $PREFIX/lib
\ No newline at end of file
+cp build/lib.linux-x86_64*/libNVCategory.so $PREFIX/lib
```

Comment:
I believe this should just be replaced with a proper `make install` now that this is open source.

Reply:
Should look basically like: https://github.com/rapidsai/cudf/blob/branch-0.6/conda/recipes/libcudf/build.sh

Reply:
This is a known issue due to using the same build process for conda & pip packages. It's on my list to fix, but not for v0.3 of nvstrings.

Reply:
👍 As long as the paths are correct to find the shared libraries then that's fine.